spark/src/main/scala/geotrellis/spark/io/slippy/SlippyTileWriter.scala

```diff
@@ -0,0 +1,32 @@
+package geotrellis.spark.io.slippy
+
+import geotrellis.vector._
+import geotrellis.raster._
+import geotrellis.raster.io.Filesystem
+import geotrellis.raster.io.geotiff._
+import geotrellis.spark._
+import geotrellis.spark.io.s3._
+import geotrellis.spark.io.hadoop._
+import geotrellis.spark.io.hadoop.formats._
+
+import org.apache.commons.io.FileUtils
+import org.apache.commons.io.filefilter._
+import org.apache.spark._
+import org.apache.spark.rdd._
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.mapreduce.Job
+import org.apache.hadoop.mapreduce.{InputSplit, TaskAttemptContext}
+import org.apache.hadoop.fs.Path
+
+import java.io.File
+import scala.collection.JavaConversions._
+
+trait TileByteWriter[T] extends Serializable {
```

Comment:
This type class is dangling, unused. Also I'm not sure it's a good interface. Kind of implies that there is only one way to write a particular T instance to bytes, when it probably depends on the extension rather than provides it. Will this have life or a dead branch?

I'll also follow up on @moradology PR to see if it makes sense there anyway.